use-cases/event-management.md

```diff
@@ -1,24 +1,85 @@
+
+
+#### Adding reservations to an availability block
+
+When a new reservation is created within the event management software, it needs to be synced with Mews, and vice versa. Reservations can be pushed into Mews using the [Add reservations](../operations/reservations.md#add-reservations) operation. To ensure that the reservations consume availability reserved for the availability block, instead of from the property's overall availability, include `AvailabilityBlockId` in the [reservations parameters](../operations/reservations..#reservation-parameters) when you [add reservations](../operations/reservations.md#add-reservations) into an existing group block.
+
+It is also possible to place an existing reservation in Mews into an availability block with the [Update reservations](../operations/reservations.md#update-reservations) operation by including `AvailabilityBlockId` in the [reservations parameters](../operations/reservations..#reservation-parameters).
+
+If applicable, you can attach a company to a reservation when calling [Add reservations](./operations/reservations.md#add-reservations) or [Update reservations](../operations/reservations.md#update-reservations).
+
+**Note:** 
+It is currently not possible to exceed the availability block capacity (the inventory allocated to an availability block through the [Update service availability](../operations/services.md#update-service-availability) operation. However, it is possible to add reservations with arrival or departure date times that extend beyond the [interval in a given availability block](../operations/services.md#availability-block-parameters). Should there be a need to accommodate more reservations in an availability block, you must first update the adjustments to increase the allocated inventory in that block.
+
+#### Managing reservations and customers
+
+To ensure that the property can further manage individual companions to the group reservation via the integration, use the [Add reservation companion](../operations/reservations.md#add-reservation-companion) or [Delete reservation companion](../operations/reservations.md#delete-reservation-companion) to assign guests to rooms.
+
+
+#### Managing availability block inventory and pickup
+
+Call [Get availability blocks](../operations/services.md#add-availability-blocks) to retrieve information about existing availability blocks, as well as all associated reservations and availability adjustments. To avoid the need of regular polling, you can make use of [Webhooks for Reservation events](../webhooks.md#general-message) to automatically receive information of reservation creation and/or changes. Note the `AvailabilityBlockId` in the [`ServiceOrders`](../webhooks.md#entities) object to record pickup for the relevant availability block in your system and in Mews.
```

Comment:
```suggestion
Call [Get availability blocks](../operations/services.md#get-all-availability-blocks) to retrieve information about existing availability blocks, as well as all associated reservations and availability adjustments. To avoid the need of regular polling, you can make use of [Webhooks for Reservation events](../webhooks.md#general-message) to automatically receive information of reservation creation and/or changes. Note the `AvailabilityBlockId` in the [`ServiceOrders`](../webhooks.md#entities) object to record pickup for the relevant availability block in your system and in Mews.
```